src/utils/api.js

```diff
@@ -0,0 +1,67 @@
+/* The API communication module.
+   Inspired by Svelte RealWorld:
+     https://github.com/sveltejs/realworld/blob/master/src/node_modules/api.js */
+
+export const API_HOST = process.env.SAPPER_APP_API_HOST;
+
+
+function request(method, url, data) {
+  const fetch = process.browser ? window.fetch : require('node-fetch').default;
+  const formDataClass = process.browser ? FormData : require('url').URLSearchParams;
+
+  let options = {
+    credentials: 'include',
+    method,
+    headers: {},
+  };
+
+  if (data instanceof formDataClass) {
+    options.body = data;
+  } else if (data != null) {
+    options.headers['Content-Type'] = 'application/json';
+    options.body = JSON.stringify(data);
+  }
+
+  return fetch(url, options)
+		.then(r => r.text())
+		.then(json => {
+			try {
+				return JSON.parse(json);
+			} catch (err) {
+				return json;
+			}
+		});
+}
+
+
+export function get(path) {
+  if (API_HOST == null) {
+    throw new Error('The API host is undefined.');
+  }
```

Comment:
Maybe this can be checked once right after declaring `API_HOST`? It's an environment variable; shouldn't be reactive

Reply:
I just wanted the server to run even without this variable (for edge debug cases where we aren't actually making requests). Idk if you think this is worth it, let me know

Reply:
Why would you need to run the server without this variable existing, not even empty? The server reads it from the .env file anyways, so you don't have to manually put it there. Also, maybe another variable like `DEBUG=1` for these cases?
Anyways, I think this is a lot of useless code repetition, preventing the function from being much simpler.